media/subtitles/it/monero-promo.vtt

```diff
@@ -0,0 +1,40 @@
+WEBVTT
+Kind: captions
+Language: it
+
+00:00:00.400 --> 00:00:06.320
+Per la maggior parte delle persone, la privacy finanziaria è molto importante, ma le banche, le grandi società e
+
+00:00:06.320 --> 00:00:11.520
+persino i governi stanno vedendo i loro documenti
+compromessi. La conclusione innegabile è che non
+
+00:00:11.520 --> 00:00:18.640
+esiste un luogo sicuro dove condurre transazioni private. Beh, non ne esisteva uno fino ad ora. Ed ecco a voi Monero.
+
+00:00:18.640 --> 00:00:25.120
+Monero è una valuta sicura, privata, non rintracciabile. Con Monero sei tu la tua banca.
+
+00:00:25.840 --> 00:00:32.560
+Solo tu controlli e sei responsabile dei tuoi
+fondi. Monero è privato, questo significa che le aziende
+
+00:00:32.560 --> 00:00:38.160
+possono tenere segreti i loro fornitori,
+mentre i cittadini sfuggono alle repressioni del governo
+
+00:00:38.160 --> 00:00:44.160
+e ai vicini ficcanaso o ai truffatori. Monero può anche essere selettivamente trasparente e puoi scegliere
+
+00:00:44.160 --> 00:00:49.920
+chi vuoi che veda le transazioni. Monero è scalabile dinamicamente, il che lo rende molto diverso
+
+00:00:49.920 --> 00:00:56.880
+da un Bitcoin. Inoltre, la ricompensa del blocco non scenderà mai al di sotto di 0,3 XMR e l'inflazione nominale
```

Comment:
```suggestion
da Bitcoin. Inoltre, la ricompensa di blocco non scenderà mai al di sotto di 0,3 XMR e l'inflazione nominale
```

See the [terminology guide for Italian](https://github.com/monero-ecosystem/monero-translations/blob/master/terminology-guides/italian-terminology.md)